src/fit.ts

```diff
@@ -68,7 +70,10 @@ export function fitTarget(Model: OdinModelConstructable,
                           data: FitData, pars: FitPars,
                           modelledSeries: string,
                           control: any) {
-    const tStart = data.time[0];
+    const tStart = 0;
+    if (data.time[0] < tStart) {
+        throw Error(`Expected the first time to be at least ${tStart}`);
```

Comment:
Should we throw an error in the app on upload data now if there are any negative times?

Reply:
yeah, that seems sensible, I've put that down as https://mrc-ide.myjetbrains.com/youtrack/issue/mrc-3615